content/en/methods/streaming.md

```diff
@@ -641,7 +648,7 @@ An example update to the public timeline:
 ```
 
 {{< hint style="warning" >}}
-Note that while the event is JSON-encoded, the `payload` is string-encoded and escaped, so it must be parsed and loaded as JSON from that string.
+Note that while the event is JSON-encoded, the `payload` is string-encoded and escaped, so it must be parsed and loaded as JSON from that string. However, for `delete` and `announcements.delete` events the payload is a string representing an identifier, not a JSON value.
```

Comment:
Make this addition into a completely separate hint after line 665 or so (Below "An example delete event from the public timeline:")

Reply:
I'm fine with this I think.